lib/Support/ErrorHandling.cpp

```diff
@@ -85,7 +92,15 @@ static void klee_vfmessage(FILE *fp, const char *pfx, const char *msg,
                        /*bg=*/false);
   }
 
-  fdos << "KLEE: ";
+  if (LogCpuTime) {
+    sys::TimeValue now(0,0), user(0,0), sys(0,0);
+    sys::Process::GetTimeUsage(now,user,sys);
+    fdos << (user.seconds() + sys.seconds()) << ".";
+    fdos << (user.nanoseconds() + sys.nanoseconds());
```

Comment:
too bad. There seems no straight-forward portable way to achieve this. I would have to call rusage twice with `RUSAGE_SELF` and `RUSAGE_CHILDREN` and sum up both.